Code/Yahoo.Yui.Compressor.Tests/TestHelpers/BuildEngine.cs

```diff
@@ -8,29 +9,34 @@ public static class BuildEngine
     {
         public static bool ContainsError(IBuildEngine engine, string error)
         {
-            return CheckForError(engine, error, true);
+            return CheckForError("Errors", engine, error, true);
         }
 
         public static bool DoesNotContainError(IBuildEngine engine, string error)
         {
-            return CheckForError(engine, error, false);
+            return CheckForError("Errors", engine, error, false);
         }
 
-        private static bool CheckForError(IBuildEngine engine, string error, bool exists)
+        public static bool ContainsMessage(IBuildEngine engine, string message)
+        {
+            return CheckForError("Messages", engine, message, true);
+        }
+
+        private static bool CheckForError(string messageType, IBuildEngine engine, string searchString, bool exists)

```

Comment:
Since the method is now checking for Errors OR Messages, then "CheckForError" is no longer an appropriate name for the method - CheckForMessage probably makes more sense since a message could be an error or a regular message.
Also ideally the messageType would be an enum since there are only two valid values